### PR TITLE
Remove Vagrant integration, use homestead instead

### LINKED ISSUE #342: Remove Vagrant integration and promote Homestead instead
Homestead supports Symfony applications. It would be very simple to get the CMF running using homestead.

Should we remove the vagrant integration from this repository and instead write a small piece of documentation about homestead?

/cc @symfony-cmf/core

### PR DESCRIPTION
**Pull request by wouterj**
Fixes #342

### CODE REVIEW

**Comment by dbu** [comment]
don't we need some sort of config file for homestead that tells it the structure and installs jackrabbit?

btw, there is a merge conflict.

**Comment by wouterj** [comment]
> don't we need some sort of config file for homestead that tells it the structure and installs jackrabbit?

No, homestead is configured on the PC. E.g. when you want to use the CMF sandbox using Homestead, all you need is this configuration in your Homestead config file (global on your PC):
```yaml
folders:
    - map: ~/sf-world/cmf/cmf-sandbox
      to: /home/vagrant/cmf-sandbox

sites:
    - map: cmf-sandbox.local
      to: /home/vagrant/cmf-sandbox/web
      type: symfony

databases: [cmf_sandbox]
```

And then configure Symfony to use the `cmf_sandbox` MySQL database.

*Fyi, merge conflict is fixed*

File: README.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ Link to the [live demo](http://sandbox.cmf.symfony.com/)
 
 ## Getting started
 
-You can run the sandbox on your system, or in a virtualbox VM using Vagrant. For the latter, see
-"Getting started using Vagrant"
+You can run the sandbox on your system if you have Apache or Nginx installed or
+[use Homestead](https://symfony.com/doc/current/setup/homestead).
```
Comment (dbu) [inline README.md]: we should explain that one wants to use the parameters file for jackalope-doctrine-dbal . i think our tutorials still assume sqlite - should we use sqlite in homestead by default?

**Comment by ElectricMaxxx** [comment]
Not sure but felt the way as if you can run it locally as I saw the SymfonyCon talk. And the you will be able to deploy in a branch alike manner. In a symfony ecosystem we should try to get a "community" license then and allow to push each forking user into "our" cloud on a unique branch. But it can be that I think to futuristic. :-)

**Pull request closed without merging**